scenarios/web/spring-boot/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```

Comment:
The guidance I could find says to check in `gradle-wrapper.jar`. I tried deleting it, and it was a pain to get back. I think the idea is that the wrapper allows you to build without an install of gradle on the machine.

Reply:
These gradle and maven scripts were generated by spring boot, I just left them alone.